benchmark/wildtime_benchmarks/data_generation_arxiv.py

```diff
@@ -1,10 +1,8 @@
 import os
 import pickle
 
-import torch
 from benchmark_utils import create_timestamp, download_if_not_exists, setup_argparser_wildtime, setup_logger
 from torch.utils.data import Dataset
-from tqdm import tqdm
```

Comment:
Why no tqdm anymore :D?

Reply:
This whole process is very quick so we don't need to keep track of how much we processed and how much is left. On the contrary the `tqdm` just adds noise to the terminal.